logbook-netty/src/test/java/org/zalando/logbook/netty/LogbookServerHandlerTest.java

```diff
@@ -33,17 +33,10 @@
 final class LogbookServerHandlerTest {
 
     private final HttpLogWriter writer = mock(HttpLogWriter.class);
+    private final Logbook logbook = Logbook.builder().strategy(new TestStrategy()).sink(new DefaultSink(new DefaultHttpLogFormatter(), writer)).build();
 
-    private final Logbook logbook = Logbook.builder()
-            .strategy(new TestStrategy())
-            .sink(new DefaultSink(new DefaultHttpLogFormatter(), writer))
-            .build();
-
-    @SuppressWarnings("deprecation") // the alternative method is not available prior to 1.0.0
     private final DisposableServer server = HttpServer.create()
-            .tcpConfiguration(tcpServer ->
-                    tcpServer.doOnConnection(connection ->
-                            connection.addHandlerLast(new LogbookServerHandler(logbook))))
+            .doOnConnection(connection -> connection.addHandlerLast(new LogbookServerHandler(logbook)))
```

Comment:
Suppression was intentional to preserve backwards compatibility.

Reply:
reverted